hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/ExpiryValidatorImpl.java

```diff
@@ -74,7 +74,7 @@ public ExpiryMeta resolveCreationAttempt(
         context.attributeValidator().validateExpiry(effectiveExpiry);
 
         // Even if the effective expiry is valid, we still also require any explicit auto-renew period to be valid
-        if (creationMeta.hasAutoRenewPeriod()) {
+        if (creationMeta.hasAutoRenewPeriod() && creationMeta.autoRenewPeriod() != 0) {
```

Comment:
Does it make sense to allow an autoRenewPeriod of 0? Maybe the calling code should have set to be `ExpiryMeta.NA` instead.

Reply:
Removed the code change to TokenCreateHandler.